controlpanel/frontend/views/user.py

```diff
@@ -36,11 +51,20 @@ def get_context_data(self, **kwargs):
                 continue
 
             try:
-                last_login = dateutil.parser.parse(user.get('last_login'))
+                auth0_last_login = user.get("last_login")
```

Comment:
I have to say this code is (not the stuff you added but the whole function) surprisingly confusing.

For example, it wasn't immediately apparent to me that this `user` wasn't the user DB record as in `models.User` but in fact the Auth0 user info. Potential suggestion => rename `users` to `auth0_users` and `user` to `auth0_user`.

The `last_logins` was also a bit surprising because it seemed like a list but then it finally got converted to a dictionary. Potential suggestion => Maybe we should just declare as an empty dictionary `{}` from the beginning and use `last_logins[auth0_id] = {...}`.

Speaking of `last_logins`, it used contain just the last logins but now we add more login-related things to it so potentially we could rename it to `login_info` (or something like that).

Reply:
I agree. The code as it was originally was very confusing to work out just what was happening.

I went for the "make it work" rather than "refactor all the things" way with this PR since I'd be refactoring into a very deep rabbit hole. If you think it useful, I'm happy to simplify and refactor.

Reply:
Up to you, I agree with you: I wouldn't go too deep into the refactoring hole but if you feel comfortable and think it's useful even rename few things in this function IMHO could help a lot the future us with readability ☺️

Reply:
:+1:

Reply:
See new comments in code. ;-)